cfgov/jinja2/v1/es/comprar-casa/index.html

```diff
@@ -1,161 +1,197 @@
 {% extends 'es/es-base.html' %}
 
 {% block babel_title %}
-	Comprar Casa &gt; Oficina para la Protección Financiera del Consumidor
+  Comprar Casa &gt; Oficina para la Protección Financiera del Consumidor
 {% endblock %}
 
 {% block body_class %}
 {% endblock %}
 
 {% block nav_item_comprar %}
-	s-active
+  s-active
 {% endblock %}
 
 {% block babel_content %}
 <header class="hero full-bleed full-bleed-padded">
-	<div class="hero-container">
-		<img class="hero-image"
-		     src="{{ static('nemo/_/img/mortgage-hero.png') }}"
-		     alt=""
-				 style="padding-left: 434px;">
-		<div class="hero-copy">
-			<h1>
-				<div class="heading-inner-small">
-					Nuestros recursos gratuitos lo<br>
-					ayudan a estar preparado.
-				</div>
-				<strong>
-					Comprar casa es una<br>
-					decisión de vida.
-				</strong>
-			</h1>
-		</div>
-	</div>
+    <div class="hero-container">
+        <img class="hero-image"
+             src="{{ static('nemo/_/img/mortgage-hero.png') }}"
+             alt=""
+         style="padding-left: 434px;">
+        <div class="hero-copy">
+            <h1>
+                <span class="heading-inner-small" style="display:block;">
```

Comment:
`style="display:block;` added here because I needed to convert the `div` to a `span`, because the former is not allowed in an `h1` at this point according to the HTML validator.